components/layout/OnThisPage.tsx

```diff
@@ -50,6 +48,7 @@ const Container = styled.div`
     box-sizing: border-box;
     border-left: 1px solid ${GREY_2};
     cursor: pointer;
+    transition: border-color 200ms ease;
```

Comment:
Nice! Love the transition. 

Could we also remove the default focus outline and add a background color like in the side-nav to the anchors in this Container? 

>  \> a {
:focus {
      outline: none;
      color: ${HEADLINE_TEXT};
      background-color: ${GREY_1};
    }
}

Reply:
Good call! Updated